libs/ngworker/lumberjack/http-driver/src/lib/errors/lumberjack-http-driver.error.ts

```diff
@@ -0,0 +1,18 @@
+export class LumberjackHttpDriverError extends Error {
+  constructor(message: string = 'LumberjackHttpDriverError') {
+    super(message);
+
+    // LumberjackHttpDriverError instanceof Error === true
+    Object.setPrototypeOf(this, new.target.prototype);
```

Comment:
question: why extending from Error does not do this?

Reply:
Must have been an old snippet from before ES classes. Removed it.